Model/Indexer/Category.php

```diff
@@ -101,10 +103,10 @@ private function rebuildAffectedProducts($storeId)
     private function processSpecificCategories($categoryIds, $categoriesPerPage, $storeId)
     {
         foreach (array_chunk($categoryIds, $categoriesPerPage) as $chunk) {
-            /** @uses Data::rebuildStoreCategoryIndex */
+            /** @uses CategoryIndexBuilder::rebuildIndexIds */
             $this->queue->addToQueue(
-                Data::class,
-                'rebuildStoreCategoryIndex',
+                CategoryIndexBuilder::class,
```

Comment:
This more clearly calls out the two different indexing ops here in this class. 👍